pkg/gui/controllers/helpers/merge_and_rebase_helper.go

```diff
@@ -149,17 +149,12 @@ func (self *MergeAndRebaseHelper) CheckMergeOrRebase(result error) error {
 		return nil
 	} else if isMergeConflictErr(result.Error()) {
 		return self.c.Confirm(types.ConfirmOpts{
-			Title:               self.c.Tr.FoundConflictsTitle,
-			Prompt:              self.c.Tr.FoundConflicts,
-			HandlersManageFocus: true,
```

Comment:
I think I missed this one when removing the other places where HandlersManageFocus proved unnecessary after changing to pop first then run the callback. I've tested this just now and it works fine